server/container/util/src/test/java/org/apache/james/util/concurrency/ConcurrentTestRunnerTest.java

```diff
@@ -132,4 +133,79 @@ public void consume(int threadNumber, int step) throws Exception {
         assertThat(concurrentTestRunner.awaitTermination(DEFAULT_AWAIT_TIME, TimeUnit.MILLISECONDS)).isTrue();
         assertThat(queue).containsOnly("0:0", "0:1", "1:0", "1:1");
     }
+
+    @Test
+    public void runShouldNotThrowOnExceptions() throws Exception {
+        int operationCount = 2;
+        int threadCount = 2;
+
```

Comment:
And just some line above you get `runShouldThrowOnExceptions` this is definitly not clear